modules/ROOT/pages/database-administration/aliases/manage-aliases-composite-databases.adoc

```diff
@@ -214,15 +214,175 @@ SHOW DATABASE garden YIELD name, type, constituents
 ----
 
 [[alias-management-escaping]]
-== Database alias names and escaping
+== Database alias names that contain dots
 
 Naming database aliases in composite databases follows the same rule as xref:database-administration/aliases/naming-aliases.adoc[naming aliases for standard databases].
 However, when it comes to escaping names using backticks, there are some additional things to consider:
 
-=== Quoting database alias and composite database names
+Dots in alias names are ambiguous.
+They could either be interpreted as part of the name itself, or as the dot that separates a composite namespace from the alias name.
+
+[role=label--new-2025.06]
+=== Conflicting names
+In versions earlier than Neo4j 2025.06, it is possible to create conflicting aliases, such as the constituent alias `flowers` within the composite database `garden` as well as the non-composite local alias `garden.flowers`.
+Both of these are referred to by the same name, `garden.flowers`.
+
+Neo4j 2025.06 and later versions ensure that no such conflicts exist and throw an exception when attempting to create a new alias with the same name as an existing alias.
+
+[.tabbed-example]
+=====
+[role=include-with-cypher-5]
+======
+Creating a regular alias with the same name as an existing composite constituent is disallowed:
+
+.Query
+[source, cypher]
+----
+CYPHER 5 CREATE COMPOSITE DATABASE `garden`
+CYPHER 5 CREATE ALIAS `garden`.`flowers` FOR DATABASE `northwind-graph`
+CYPHER 5 CREATE ALIAS `garden.flowers` FOR DATABASE `northwind-graph`
+----
+
+.Error message
+[source, output, role="noheader"]
+----
+Failed to create the specified database alias 'garden.flowers': Database name or alias already exists.
+----
+The exception has the GQLSTATUS code link:{neo4j-docs-base-uri}/status-codes/current/errors/gql-errors/42001[42001 - invalid syntax] with the cause link:{neo4j-docs-base-uri}/status-codes/current/errors/gql-errors/42N11[42N11 - graph reference already exists].
+
+======
+
+[role=include-with-cypher-25]
+======
+
+Creating a regular alias with the same name as an existing composite constituent is disallowed.
+In Cypher 25, the two names are considered the same since the parts are not quoted separately as they would be in Cypher 5.
+
+.Query
+[source, cypher]
+----
+CYPHER 25 CREATE COMPOSITE DATABASE `garden`
+CYPHER 25 CREATE ALIAS `garden.flowers` FOR DATABASE `northwind-graph`
+CYPHER 25 CREATE ALIAS `garden.flowers` FOR DATABASE `northwind-graph`
```

Comment:
shouldn't one of these be un escaped to show some difference in the commands?

Reply:
Sure, @l-heemann can confirm if that is something we want to update when he gets back :P

Reply:
Yes this is intentional to show the difference between Cypher 5 and 25 (by toggling the tabbed view back and forth)

Reply:
But having both the CYPHER 25 commands be identical feels weird to me... if I did that in Cypher 5 I would fail on it trying to create the same thing twice as well... so why not have one ` garden.flowers ` (unescaped) and one ``` `garden.flowers` ``` (escaped together) in the Cypher 25 example 🤔 wouldn't that be closer to the commands for Cypher 5 (that does differ in that one is escaped together and one separately)